### PR TITLE
Questions endpoint

### PR DESCRIPTION
**Pull request by ewsams**
made changes to the self-tags component  and enviroment.ts file.

### CODE REVIEW

File: src/app/experts/self-tags/self-tags.component.ts [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+    
+    constructor(private tagsService: TagService) { }
+
 
-  onSubmit(f: NgForm) {
-    return true;
-  }
-  constructor() { }
+    ngOnInit() {
+      this.tagsService.getTags().subscribe(tags => {
```
Comment (AdamDKing) [inline src/app/experts/self-tags/self-tags.component.ts]: tagsService -> tagService

File: src/app/models/Tags.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export interface Tags {
+    id: number;
+    tagName: string;
```
Comment (AdamDKing) [inline src/app/models/Tags.ts]: too big indent, get prettier!

File: src/app/experts/self-tags/self-tags.component.ts [hunk 3]
```diff
@@ -1,25 +1,28 @@
 import { Component, OnInit } from '@angular/core';
-import { NgForm } from '@angular/forms';
-import { HttpClient } from '@angular/common/http';
-
+import {Tags} from '../../models/Tags';
+import { TagService } from 'src/app/services/tags.service';
```
Comment (AdamDKing) [inline src/app/experts/self-tags/self-tags.component.ts]: indenting in this file needs to be fixed

File: src/app/services/tags.service.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { Tags } from '../models/Tags';
+import {environment} from '../../environments/environment';
+
```
Comment (AdamDKing) [inline src/app/services/tags.service.ts:6]: indent in this file needs to be fixed

**Review by AdamDKing — CHANGES_REQUESTED** [review]
Some formatting changes and a small naming change.  Make sure to get and use Prettier

File: src/app/experts/self-tags/self-tags.component.ts (hunk 1 above)
Comment (ewsams) [reply src/app/experts/self-tags/self-tags.component.ts]: naming convention resolved.

File: src/app/models/Tags.ts (hunk 2 above)
Comment (ewsams) [reply src/app/models/Tags.ts]: used the alt shift F command to correct.

File: src/app/experts/self-tags/self-tags.component.ts (hunk 3 above)
Comment (ewsams) [reply src/app/experts/self-tags/self-tags.component.ts]: used the alt shift f command to make changes.

File: src/app/services/tags.service.ts (hunk 4 above)
Comment (ewsams) [reply src/app/services/tags.service.ts:6]: used shift alt F command to make changes

**Comment by ewsams** [comment]
I believe I have made the necessary formatting changes.

**Review by AdamDKing — CHANGES_REQUESTED** [review]
fix the single-to-double quote formatting.  Change those double quotes back to single quotes

**Pull request merged**